File: Api/Models/ToDo.cs
using System.ComponentModel;

namespace TravelPlannerApp.Models
{
    public class ToDo
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public Country? Country { get; set; }
        public City City { get; set; }
        [DefaultValue(0)]
        public int SortOrder { get; set; } = 0;
    }
}

File: website/src/app/models/todo.ts
import { City } from "./city";
import { Country } from "./country";

export interface ToDo {
    id:number;
    name : string;    
    city: City;
    country: Country;
    sortOrder: number;
}

export interface ToDoUpdate{
    id: number; 
    name: string;
    cityId: number;
    countryId: number;
    sortOrder:number;
}